env-dist

```diff
@@ -24,10 +24,7 @@ OIDC_OP_STORE_ACCESS_TOKEN = True
 # Mozillians
 MOZILLIANS_API_URL = https://mozillians.org/api/v2/
 # Default database setting for docker setup
-DATABASE_URL = 'mysql://remo:remo@db:3306/remo'
-# Uncomment the below DATABASE_URL setting if you're running the setup
-# locally using virtualenv
-# DATABASE_URL = 'mysql://remo:remo@localhost:3306/remo'
+DATABASE_URL = 'postgres://remo:remo@db:5432/remo'
```

Comment:
won't this break virtualenv setups? I think we should keep the comment at least

Reply:
Do we need to maintain the virtualenv setup ?

Reply:
Given that the Reps Portal will be decommissioned at some point, maybe not. However then we should also remove the docs for it.